.travis.yml

```diff
@@ -30,8 +30,7 @@ before_script:
   - kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/v$K6T_VER/kubevirt.yaml
   - k_wait_all_running pods
 
-script:
-  # Deploy manifests
+  # Deploy CDI
```

Comment:
Note: the secret manifest yaml may not be usable as-is since it has no keys defined. If you don't need keys then I don't think you need this secret at all (of course don't refer to it in the pvc).